src/game/boe.main.cpp

```diff
@@ -257,11 +259,15 @@ void replay_next_action() {
 
 	auto next_action = pop_next_action();
 	std::string t = next_action->Value();
-	if(t == "load_party") {
+	
+	if(overall_mode == MODE_STARTUP && t == "startup_button_click"){
+		eStartButton btn = static_cast<eStartButton>(atoi(next_action->GetText().c_str()));
```

Comment:
I'm not going to consider this a blocker, but I think it would be better to use the `estreams.cpp` conversion mechanism here, giving a string representation for each enum value.